From a0f8e748159b6bc0b8e5eaa1f9eb58a18ca736fa Mon Sep 17 00:00:00 2001
From: David Edmundson <kde@davidedmundson.co.uk>
Date: Thu, 13 Feb 2020 16:52:40 +0000
Subject: Sync IndexerConfig on exit

Summary:
KCoreConfigSkeleton, unlike KConfig does not automagically save on exit.

BUG: 417127

Test Plan:
balooctl disable
balooctl enable

Subscribers: kde-frameworks-devel, #baloo

Tags: #frameworks, #baloo

Differential Revision: https://phabricator.kde.org/D27376
---
 src/lib/indexerconfig.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/lib/indexerconfig.cpp b/src/lib/indexerconfig.cpp
index 66786b9..19ae342 100644
--- a/src/lib/indexerconfig.cpp
+++ b/src/lib/indexerconfig.cpp
@@ -43,6 +43,7 @@ IndexerConfig::IndexerConfig()
 
 IndexerConfig::~IndexerConfig()
 {
+    d->m_settings.save();
     delete d;
 }
 
-- 
cgit v1.1
